code/controllers/subsystem/mapping.dm

```diff
@@ -251,7 +251,10 @@ SUBSYSTEM_DEF(mapping)
 	// Generate deep space ruins
 	var/list/space_ruins = levels_by_trait(ZTRAIT_SPACE_RUINS)
 	if (space_ruins.len)
-		seedRuins(space_ruins, CONFIG_GET(number/space_budget), list(/area/space), themed_ruins[ZTRAIT_SPACE_RUINS])
+		// Create a proportional budget by multiplying the amount of space ruin levels in the current map over the default amount
+		var/default_ruin_amount = 7
```

Comment:
this should be a local define, `#define DEFAULT_RUIN_AMOUNT 7` instead of a var here IMO, since there's no need to have it be a variable for all intents and purposes here. Remember to `undef` at the end of the file

Reply:
Shouldn't it be a global define since it has to be the same value as var/space_ruin_levels in map_config.dm? I just set it to 7 because I couldn't figure out the correct method for using the same variable as another object, but global define seems like it would work.

Reply:
i mean if it's like that then sure